content/en/docs/concepts/architecture/cri.md

```diff
@@ -29,10 +29,10 @@ runtime, which can be configured separately within the kubelet by using the
 `--image-service-endpoint` and `--container-runtime-endpoint` [command line
 flags](/docs/reference/command-line-tools-reference/kubelet)
 
-For Kubernetes v{{< skew currentVersion >}}, the kubelet prefers to use CRI `v1`.
+For Kubernetes v{{< skew currentPatchVersion >}}, the kubelet prefers to use CRI `v1`.
 If a container runtime does not support `v1` of the CRI, then the kubelet tries to
 negotiate any older supported version.
-The v{{< skew currentVersion >}} kubelet can also negotiate CRI `v1alpha2`, but
+The v{{< skew currentPatchVersion >}} kubelet can also negotiate CRI `v1alpha2`, but
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.